pkgs/by-name/pr/proton-mail/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, dpkg
+, electron
+}
+:
+stdenv.mkDerivation rec {
+  pname = "proton-mail";
+  versionBase = "1.0.0";
+  versionSuffix = "beta";
```

Comment:
Why do we need a suffix?

Reply:
we don't need it. the verison of the .deb is 1.0.0 but the "beta" is implied in the release notes and in the description. I thought it would be fair to the users to know that this is in fact still in beta. But if you don't think it's needed, I can make this just version = "1.0.0". No strong feelings here

Reply:
We should use the version used by upstream, not come up with our own versioning scheme.